# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../entrance/x11r7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- entrance/src/daemon/Entranced.h.orig	2000-01-01 00:00:00.000000000 +0100
+++ entrance/src/daemon/Entranced.h	2005-11-15 20:03:20.000000000 +0100
@@ -19,8 +19,8 @@
 
 #define ENTRANCED_DEBUG 1
 
-#define X_SERVER "/usr/X11R6/bin/X -quiet -nolisten tcp"
-//#define X_SERVER "/usr/X11R6/bin/Xnest -full"
+#define X_SERVER "/usr/X11R7/bin/X -quiet -nolisten tcp"
+//#define X_SERVER "/usr/X11R7/bin/Xnest -full"
 #define X_DISP ":0"             /* only used if DISPLAY variable is NOT set */
 #define ENTRANCE PREFIX "/bin/entrance_wrapper"
 #define PIDFILE "/var/run/entranced.pid"
